app/assets/stylesheets/home.scss

```diff
@@ -1,14 +1,31 @@
-.curious {
-  background: asset-url("curiousity.jpg") no-repeat center center scroll;
-  background-size: cover;
+.columns {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+.about {
+  padding: 2rem;
+  background-color: #d5ffea;
+  border-radius: 50px;
+}
+
+.example-chart {
+  background-color: #d5ffea;
+  border-radius: 50px;
+
+  a {
+    color: black;
+    margin-top: 1rem;
+  }
+  a:hover {
+    color: #1585cf;
```

Comment:
Color literals like `#1585cf` should only be used in variable declarations; they should be referred to via variable everywhere else.